goplugin/mw_go_plugin_test.go

```diff
@@ -1,6 +1,8 @@
 // +build goplugin
 
-package goplugin_test
+//go:generate go build -buildmode=plugin -o ../test/goplugins/goplugins.so ../test/goplugins/test_goplugin.go
+
+package goplugin
```

Comment:
@buger This package name change is causing import cycle error. Do you have any chance to look at it?